### DISCUSSION TITLE
Error: PrismaClient is unable to be run in the browser.

### DISCUSSION THREAD

**Discussion by sowai00** [prisma/prisma#21826] [Q&A]
### Bug description
```
Unhandled Runtime Error
lib/prisma.ts (6:39) @ eval
  1  | import { PrismaClient } from "@prisma/client";
  4 | var prisma: PrismaClient | undefined;
  5 | }
> 6 | export const prisma = global.prisma || new PrismaClient();
    |                                     ^
  7 | 
  8 | if (process.env.NODE_ENV !== "production") global.prisma = prisma;
  9 | 
```
### How to reproduce
```ts
import { prisma } from "../../lib/prisma";
export async function GetStaticProps(context: GetStaticPropsContext) {
  const notes = await prisma.note.findMany({
    select: {
      title: true,
      id: true,
      content: true,
    },
  });
  return {
    props: {
      notes,
    },
  };
}
```

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS:  macOS
- Database: MongoDB
- Node.js version: v16.15.0

### Prisma Version

```
"@prisma/client": "^4.16.2",
```

**Comment by ludralph** [comment]
Hi @sowai00 👋 

Can you please take a look at this [issue](https://github.com/prisma/prisma/issues/6219). It has some solutions to this same problem. Also, you might want to look at this [documentation](https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices) for instantiating Prisma Client with Next.js

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
